### PR TITLE
Fixed Unittests on MacOS

### PR DESCRIPTION
**Pull request by linuxunil**
I added some OS detection to the unittests so that they will compare the right escape sequences.  If someone could verify that the sequences I'm checking against are correct that would be awesome.  This should be able to close #103

### CODE REVIEW

File: tests/test_sequences.py [hunk 1]
```diff
@@ -16,31 +17,33 @@ def test_set_special(self):
         """> Create special escape sequence."""
         util.Color.alpha_num = 100
         result = sequences.set_special(11, COLORS["special"]["background"])
-        self.assertEqual(result, "\033]11;#1F211E\007")
+        if platform.uname()[0] == "Darwin":
```
Comment (dylanaraps) [inline tests/test_sequences.py]: Add a line break above this.

File: tests/test_sequences.py [hunk 2]
```diff
@@ -16,31 +17,33 @@ def test_set_special(self):
...
+            self.assertEqual(result, "\033]Ph1F211E\033\\")
+        else:
+            self.assertEqual(result, "\033]11;#1F211E\007")
 
     def test_set_special_alpha(self):
         """> Create special escape sequence with alpha."""
         util.Color.alpha_num = 99
         result = sequences.set_special(11, COLORS["special"]["background"])
-        self.assertEqual(result, "\033]11;[99]#1F211E\007")
+        if platform.uname()[0] == "Darwin":
```
Comment (dylanaraps) [inline tests/test_sequences.py]: Add a line break above this.

File: tests/test_sequences.py [hunk 3]
```diff
@@ -16,31 +17,33 @@ def test_set_special(self):
...
+            self.assertEqual(result, "\033]Ph1F211E\033\\")
+        else:
+            self.assertEqual(result, "\033]11;[99]#1F211E\007")
 
     def test_set_color(self):
         """> Create color escape sequence."""
         result = sequences.set_color(11, COLORS["colors"]["color0"])
-        self.assertEqual(result, "\033]4;11;#1F211E\007")
+        if platform.uname()[0] == "Darwin":
```
Comment (dylanaraps) [inline tests/test_sequences.py]: Add a line break above this.

**Comment by linuxunil** [comment]
Made the changes.  And I promise I'm reading the whole Pro Git book today so my commit logs aren't such garbage.

**Comment by dylanaraps** [comment]
Lol, it’s all good!

**Comment by linuxunil** [comment]
Off topic for a minute have you thought about setting up a chat or mailing
list for pywal.  Might cut down on some of the troubleshooting through
issues and allow talk about design.

**Comment by dylanaraps** [comment]
I think that the issue tracker here is sufficient. I'm happy for troubleshooting, discussion or anything to occur since `pywal` doesn't get that many bug reports.

**Pull request merged**